XamlControlsGallery/ControlPages/ToggleSwitchPage.xaml

```diff
@@ -35,9 +35,13 @@
             </StackPanel>
             <local:ControlExample.Xaml>
                 <x:String>
-                    &lt;ToggleSwitch Header="Toggle work" OffContent="Do work" OnContent="Working" IsOn="$(isOn)$(isOff)" /&gt;
+                    &lt;StackPanel Orientation=&quot;Horizontal&quot;&gt;
+                    &lt;ToggleSwitch Header=&quot;Toggle work&quot; OffContent=&quot;Do work&quot; OnContent=&quot;Working&quot; IsOn=&quot;$(isOn)$(isOff)&quot; /&gt;
+                    &lt;ProgressRing IsActive=&quot;{x:Bind ToggleSwitch2.IsOn, Mode=OneWay}&quot; Width=&quot;130&quot;/&gt;
+                    &lt;/StackPanel&gt;
```

Comment:
The code sample is not correctly formatted in the app: 
![Image shown ill formatted code](https://user-images.githubusercontent.com/16122379/103183965-d123d900-48b5-11eb-8104-35e22997aff8.png)

You can get the code to be correctly formatted by adding `xml:space="preserve"` to the x:String in line 37 and manually formatting the text then. You can find an example how to "format" this here: https://github.com/microsoft/Xaml-Controls-Gallery/blob/7faea5a5cb74a9b9db7c6a7d291f9b890ce06420/XamlControlsGallery/ControlPages/AppBarSeparatorPage.xaml#L33-L41

The only important thing is adding the xml:preserver and that the formatted code then starts at the left. Thank you for your PR!

Reply:
I couldn't really see it as it would be in the app, because there stil was a little issue when I tried to debug/run it here.
This doesn't address the "Related controls" section, but that's a different subject and I'll leave that up to the colleague you already mentioned.

Reply:
Oh hmm, you shouldn't have problems running the app when freshly cloned. Do you remember what the issue was, maybe we can fix it or find a workaround.

Reply:
The problem was missing (.exe) files in a folder
![image](https://user-images.githubusercontent.com/65828559/103184581-13024e80-48b9-11eb-8008-85063c871769.png)

All the folders `Debug` and `Release` (inside `ARM`, `x64`, and `x86`) are empty. Probably just a "local" cause?

Reply:
That's weird. Have you tried rebuilding the solution? If you try to try to debug the app on your machine, it should build the app; maybe that didn't work correctly? Though I have to say that I haven't seen this message before, the fact that the output folders seem to be empty is definitely unusual.

Reply:
LOL, I was using the thing on the single xaml file instead of the project. My bad. Never mind 🤐 `#StillLearning`